sessions/js-createelement/social-media-post/css/styles.css

```diff
@@ -0,0 +1,54 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
```

Comment:
Compared to the other exercise, we are a bit inconsistent with the base styles. For the `product-teaser` we added a margin and padding reset. Here it is not set. It might be nice to style consistently?

Reply:
Good catch. I adjusted the exercise